web/pdf_sidebar.js

```diff
@@ -454,6 +460,81 @@ class PDFSidebar {
       }
     });
   }
+
+  resize(width) {
+    if (width < 200 || width > this.mainContainer.offsetWidth) {
+      return;
+    }
+
+    // This is the media query in viewer.css, which
+    // sets #mainContainer left to 0 on small screens.
+    // We don't want to override that.
+    var mql = window.matchMedia('all and (max-width: 840px');
```

Comment:
Per http://caniuse.com/#search=matchMedia most of browsers we are supporting are not mentioned.